nixos/modules/services/networking/webhook.nix

```diff
@@ -0,0 +1,189 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.webhook;
+  defaultUser = "webhook";
+
+  hooks-file = pkgs.writeText "webhook-config" (
```

Comment:
The naming convention for variables, option names and possibly other things is `camelCase`. Package names are a notable exception.
Could you also rename the options to camel case?